lib/features/mouse/Mouse.js

```diff
@@ -0,0 +1,60 @@
+export default function Mouse(eventBus) {
+  var self = this;
+
+  this._lastMoveEvent = null;
+
+  function setLastMoveEvent(mousemoveEvent) {
+    self._lastMoveEvent = mousemoveEvent;
+  }
+
+  eventBus.on('canvas.init', function(context) {
+    var svg = self._svg = context.svg;
+
+    svg.addEventListener('mousemove', setLastMoveEvent);
+  });
+
+  eventBus.on('canvas.destroy', function() {
+    self._lastMouseEvent = null;
+
+    self._svg.removeEventListener('mousemove', setLastMoveEvent);
+  });
+}
+
+Mouse.$inject = [ 'eventBus' ];
+
+Mouse.prototype.getLastMoveEvent = function() {
+  return this._lastMoveEvent || createMoveEvent(0, 0);
+};
+
+// helpers //////////
+
+export function createMoveEvent(x, y) {

```

Comment:
Cool stuff, maybe we can use it to improve the situation in the dirty [`HoverFixSpec` ](https://github.com/bpmn-io/diagram-js/blob/develop/test/spec/features/dragging/HoverFixSpec.js#L54). 👍